libmscore/unrollrepeats.cpp

```diff
@@ -0,0 +1,196 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2019 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+/**
+ File handling: loading and saving.
+ */
+
+#include "xml.h"
+#include "element.h"
+#include "note.h"
+#include "chord.h"
+#include "rest.h"
+#include "sig.h"
+#include "clef.h"
+#include "key.h"
+#include "score.h"
+#include "page.h"
+#include "dynamic.h"
+#include "style.h"
+#include "tempo.h"
+#include "select.h"
+#include "staff.h"
+#include "part.h"
+#include "utils.h"
+#include "barline.h"
+#include "slur.h"
+#include "hairpin.h"
+#include "ottava.h"
+#include "textline.h"
+#include "pedal.h"
+#include "trill.h"
+#include "volta.h"
+#include "timesig.h"
+#include "box.h"
+#include "excerpt.h"
+#include "system.h"
+#include "tuplet.h"
+#include "keysig.h"
+#include "measure.h"
+#include "undo.h"
+#include "repeatlist.h"
+#include "beam.h"
+#include "stafftype.h"
+#include "revisions.h"
+#include "lyrics.h"
+#include "segment.h"
+#include "tempotext.h"
+#include "sym.h"
+#include "image.h"
+#include "tiemap.h"
+#include "tie.h"
+#include "measurebase.h"
+#include "chordlist.h"
+#include "mscore.h"
+
+
+namespace Ms {
+
+void removeRepeatMarkings(Score* score)
```

Comment:
If these functions are not members of any class and they are used only within this file you may want to declare them `static`, like here:
https://github.com/musescore/MuseScore/blob/0c4aefdd6c5bf49eb7b92c36ffa7bd858296174f/libmscore/layout.cpp#L3169

Not strictly necessary, of course.